components/equihash/src/lib.rs

```diff
@@ -26,3 +26,6 @@ mod verify;
 mod test_vectors;
 
 pub use verify::{is_valid_solution, Error};
+
+mod blake2b;
+pub mod tromp;
```

Comment:
TODO: Gate this off behind a feature flag, so anyone who doesn't need it won't need to depend on `cc` or include any of the native code.

Reply:
I implemented this feature (and got the code producing valid solutions) in PR #1088.